ios/browser/api/ledger/brave_ledger.mm

```diff
@@ -984,9 +984,7 @@ - (void)setMinimumNumberOfVisits:(int)minimumNumberOfVisits {
 }
 
 - (void)setAllowUnverifiedPublishers:(bool)allowUnverifiedPublishers {
```

Comment:
Can we remove this function and its references?

Reply:
Yeah, but now it's a no-op. 🙂

Reply:
@kylehickinson can remove it for us, I think. 🙂

Reply:
Doesn't it need to be here to not break iOS, though?

Reply:
It would break iOS temporarily, but I think this is how we did it last time I had to remove something similar.

Reply:
I prefer not to do it that way.

Reply:
Done